start.exs

```diff
@@ -0,0 +1,2 @@
+{:ok, _sup, _pipeline} = Membrane.Pipeline.start_link(Basic.Pipeline)
+Process.sleep(500)
```

Comment:
We can wait until the pipeline terminates.
You can start monitoring the pipeline and then wait until `:DOWN` message is delievered (example: https://github.com/membraneframework/membrane_demo/blob/c68406e17914aba5d7dbd403e8e04e5aa7d9de77/mix_audio/mix_audio.exs#L62)

Reply:
The pipeline fails to stop using that approach, I guess something needs to handle pipeline termination when stream ends?